### DISCUSSION TITLE
Welcome to ros2_planning_system Discussions!

### DISCUSSION THREAD

**Discussion by fmrico** [PlanSys2/ros2_planning_system#130] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avekarpp** [comment]
Hello!
I am Pranav Avekar, a Master's Student working to develop a completely automated system based on PDDL and ROS systems for autonomous robots in different domains.
 
I need guidance. How can we include the 'problem.pddl' file along with 'domain.pddl' in `launch_arguments` to bypass the use of the terminal in order to automate the complete process?

Also, any documentation related to the difference between  `plansys2_bringup_launch_monolithic` and `plansys2_bringup_launch_distributed` would be helpful.

Thank you!

Best Regards,
Pranav

**Reply by fmrico** [reply]
#200 added something similar, but it is not exactly loading a problem in pddl.

Maybe it is something interesting to implement :thinking: 

Best
Francisco

**Reply by avekarpp** [reply]
@fmrico, 
I appreciate your support. I will look to figure out a way to solve this and will let you know.

Regards,
Pranav

**Comment by ffff8240** [comment]
Hi, I'm new to all this and trying to learn. After reading the documentation for BehaviorTree.CPP I found that it's not the current version that is used in PlanSys2. Is there a plan to move to BehaviorTree.CPP v4?

**Comment by fmrico** [comment]
Hi @ffff8240 

Yes, there are plans, but I must study hard to make the most of the new version. Do you want to help on this?

Best

**Comment by robodrome** [comment]
Hi this robodrome (david). I am looking into PlanSys2 for adding autonomy to mobile platform(s).
